js/renovation/ui/scroll_view/scroll_view.tsx

```diff
@@ -38,6 +38,7 @@ export const viewFunction = (viewModel: ScrollView): JSX.Element => {
       {...props}
       // eslint-disable-next-line react/jsx-props-no-spreading
       {...restAttributes}
+      needScrollViewWrappers
```

Comment:
Where it is initialized to 'true' to work like old code?
```
    _initContent: function() {
        const $content = $('<div>').addClass(SCROLLVIEW_CONTENT_CLASS);
        this._$content.wrapInner($content);
    },
```

Reply:
These changes are consistent. It's value already is true